src/newconnection.cpp

```diff
@@ -0,0 +1,14 @@
+#include "newconnection.h"
+#include "ui_newconnection.h"
+
+newconnection::newconnection(QWidget *parent) :
```

Comment:
Would you mind renaming `newconnection` to `NewConnection`? Just for the sake of having a single way of naming things. Classes, methods: PascalCase, variables, class properties: snake_case